background.js

```diff
@@ -109,7 +125,7 @@ function sendTabToEndOfDeque(windowId, tabId) {
 
     removeFromDeque(tabId, currentDeque);
     currentDeque.push(tabId);
-    browser.tabs.update(currentDeque[0], {active: true});
+    browser.tabs.update(currentDeque[0], { active: true });
```

Comment:
Can you please revert these whitespace changes? My bad really because I don't have eslint setup to check the formatting. I will do that, but I don't have time just now.